### ISSUE TITLE
CI (travis) tests failed for Python API client

### ISSUE THREAD

**Issue by wing328** [swagger-api/swagger-codegen#4637]
##### Description

CI tests failed with the following error message:
```
Installed /home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.venv/lib/python2.7/site-packages/urllib3-1.20-py2.7.egg
Finished processing dependencies for petstore-api==1.0.0
GLOB sdist-make: /home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/setup.py
py27 create: /home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27
py27 installdeps: -r/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/requirements.txt, -r/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/test-requirements.txt
py27 inst: /home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/dist/petstore_api-1.0.0.zip
ERROR: invocation failed (exit code 1), logfile: /home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/log/py27-2.log
ERROR: actionid: py27
msg: installpkg
cmdargs: ['/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/bin/pip', 'install', '/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/dist/petstore_api-1.0.0.zip']
env: {'TRAVIS_PULL_REQUEST_BRANCH': '', 'rvm_version': '1.27.0 (latest)', 'LC_CTYPE': 'en_US.UTF-8', 'TRAVIS': 'true', 'GIMME_ENV': '/home/travis/.gimme/envs/go1.4.env', 'TRAVIS_REPO_SLUG': 'swagger-api/swagger-codegen', 'TRAVIS_STACK_LANGUAGES': '__sugilite__ android c c++ clojure cplusplus cpp crystal csharp d dart default elixir erlang go groovy haskell haxe hhvm java julia legacy mega node_js perl perl6 php pure_java python r ruby rust scala universe', 'JRUBY_OPTS': ' --client -J-XX:+TieredCompilation -J-XX:TieredStopAtLevel=1 -Xcext.enabled=false -J-Xss2m -Xcompile.invokedynamic=false', 'VIRTUAL_ENV': '/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27', 'SHELL': '/bin/bash', 'PYENV_SHELL': 'bash', 'TRAVIS_BRANCH': 'master', 'TRAVIS_PULL_REQUEST_SLUG': '', 'HISTSIZE': '1000', 'MAVEN_OPTS': '-Dmaven.repo.local=/home/travis/.m2/repository -Xmx384m -XX:MaxPermSize=192m', 'RBENV_SHELL': 'bash', 'VIRTUAL_ENV_DISABLE_PROMPT': 'true', 'rvm_remote_server_url4': 'https://s3.amazonaws.com/travis-rubies/binaries', 'MANPATH': '/home/travis/.nvm/versions/node/v4.1.2/share/man:/usr/local/clang-3.5.0/share/man:/usr/local/man:/usr/local/share/man:/usr/share/man', 'JAVA_HOME': '/usr/lib/jvm/java-7-oracle', 'MFLAGS': '', 'XDG_RUNTIME_DIR': '/run/user/1000', 'J2SDKDIR': '/usr/lib/jvm/java-9-oracle', '_system_type': 'Linux', 'TRAVIS_SECURE_ENV_VARS': 'true', 'MY_RUBY_HOME': '/home/travis/.rvm/rubies/ruby-2.3.1', 'XDG_SESSION_ID': '2', 'RUBY_VERSION': 'ruby-2.3.1', '_system_version': '14.04', 'TRAVIS_COMMIT_RANGE': '5877ceb43ee6...014cfe746b50', 'MAIL': '/var/mail/travis', 'SSH_CONNECTION': '162.222.183.179 60489 10.10.8.167 22', 'DOCKER_HUB_EMAIL': 'fehguy@gmail.com', 'GOPATH': '/home/travis/build/swagger-api/swagger-codegen/Godeps/_workspace', 'CONTINUOUS_INTEGRATION': 'true', 'GOROOT': '/home/travis/.gimme/versions/go1.4.linux.amd64', 'TRAVIS_STACK_TIMESTAMP': '2016-12-05 18:01:17 UTC', 'RACK_ENV': 'test', 'USER': 'travis', 'NVM_IOJS_ORG_MIRROR': 'https://iojs.org/dist', 'PS4': '+', 'MAKELEVEL': '1', 'SHLVL': '3', 'TRAVIS_PULL_REQUEST_SHA': '', 'MERB_ENV': 'test', 'JDK_SWITCHER_DEFAULT': 'oraclejdk8', 'GIT_ASKPASS': 'echo', 'rvm_binary_flag': '1', 'M2_HOME': '/usr/local/maven', 'DOCKER_HUB_PASSWORD': 'Bacon1DO', 'GEM_PATH': '/home/travis/.rvm/gems/ruby-2.3.1:/home/travis/.rvm/gems/ruby-2.3.1@global', 'HAS_ANTARES_THREE_LITTLE_FRONZIES_BADGE': 'true', 'TRAVIS_EVENT_TYPE': 'push', 'rvm_remote_server_verify_downloads4': '1', 'TRAVIS_TEST_RESULT': '0', '_': '/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.venv/bin/tox', 'TRAVIS_TAG': '', 'TRAVIS_BUILD_NUMBER': '7112', 'TRAVIS_STACK_FEATURES': 'basic chromium disabled-ipv6 docker docker-compose firefox go-toolchain google-chrome jdk memcached mongodb mysql nodejs_interpreter perl_interpreter perlbrew phantomjs postgresql python_interpreter rabbitmq redis ruby_interpreter sphinxsearch sqlite xserver', '_system_name': 'Ubuntu', 'PAGER': 'cat', 'TRAVIS_SUDO': 'true', 'MIX_ARCHIVES': '/home/travis/.kiex/mix/elixir-1.0.4', 'TRAVIS_BUILD_ID': '194540405', 'NVM_NODEJS_ORG_MIRROR': 'https://nodejs.org/dist', 'NVM_DIR': '/home/travis/.nvm', 'LC_ALL': 'en_US.UTF-8', 'HOME': '/home/travis', 'TRAVIS_PULL_REQUEST': 'false', 'LANG': 'en_US.UTF-8', 'NVM_BIN': '/home/travis/.nvm/versions/node/v4.1.2/bin', 'TRAVIS_COMMIT': '014cfe746b50492fed123e7071f3a0e273a270fe', 'NVM_PATH': '/home/travis/.nvm/versions/node/v4.1.2/lib/node', 'PYTHONHASHSEED': '3710334321', 'TRAVIS_STACK_JOB_BOARD_REGISTER': '/.job-board-register.yml', '_system_arch': 'x86_64', 'MYSQL_UNIX_PORT': '/var/run/mysqld/mysqld.sock', 'CI': 'true', 'rvm_prefix': '/home/travis', 'TRAVIS_JDK_VERSION': 'oraclejdk7', 'DEBIAN_FRONTEND': 'noninteractive', 'IRBRC': '/home/travis/.rvm/rubies/ruby-2.3.1/.irbrc', 'rvm_path': '/home/travis/.rvm', 'CASHER_DIR': '/home/travis/.casher', 'NLSPATH': '/usr/dt/lib/nls/msg/%L/%N.cat', 'TRAVIS_STACK_NODE_ATTRIBUTES': '/.node-attributes.yml', 'SSH_TTY': '/dev/pts/1', 'DOCKER_IMAGE_NAME': 'swaggerapi/swagger-generator', 'PERLBREW_HOME': '/home/travis/.perlbrew', 'J2REDIR': '/usr/lib/jvm/java-9-oracle', 'GEM_HOME': '/home/travis/.rvm/gems/ruby-2.3.1', 'HAS_JOSH_K_SEAL_OF_APPROVAL': 'true', 'PYTHON_CFLAGS': '-g -fstack-protector --param=ssp-buffer-size=4 -Wformat -Werror=format-security', 'COMPOSER_NO_INTERACTION': '1', 'NVM_CD_FLAGS': '', 'SSH_CLIENT': '162.222.183.179 60489 22', 'PERLBREW_BASHRC_VERSION': '0.78', 'LOGNAME': 'travis', 'PATH': '/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/bin:/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.venv/bin:/home/travis/build/swagger-api/swagger-codegen/Godeps/_workspace/bin:/home/travis/.gimme/versions/go1.4.linux.amd64/bin:/home/travis/.rvm/gems/ruby-2.3.1/bin:/home/travis/.rvm/gems/ruby-2.3.1@global/bin:/home/travis/.rvm/rubies/ruby-2.3.1/bin:/home/travis/.rvm/bin:/home/travis/bin:/home/travis/.local/bin:/home/travis/.pyenv/shims:/home/travis/.phpenv/shims:/home/travis/gopath/bin:/home/travis/.gimme/versions/go1.7.linux.amd64/bin:/home/travis/.local/bin:/opt/pyenv/bin:/opt/python/2.7.12/bin:/opt/python/3.5.2/bin:/opt/python/2.6.9/bin:/opt/python/3.2.6/bin:/opt/python/3.3.6/bin:/opt/python/3.4.4/bin:/opt/python/pypy-5.4.1/bin:/opt/python/pypy3-2.4.0/bin:/usr/local/phantomjs/bin:/usr/local/phantomjs:/home/travis/perl5/perlbrew/bin:/home/travis/.nvm/versions/node/v4.1.2/bin:/usr/local/maven-3.1.1/bin:/home/travis/.kiex/elixirs/elixir-1.0.4/bin:/home/travis/.kiex/bin:/usr/local/gradle/bin:/usr/local/clang-3.5.0/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/lib/jvm/java-9-oracle/bin:/usr/lib/jvm/java-9-oracle/db/bin:/home/travis/.phpenv/bin', 'rvm_silence_path_mismatch_check_flag': '1', 'rvm_autoupdate_flag': '0', 'MAKEFLAGS': '', 'TERM': 'xterm', 'TZ': 'UTC', 'HISTFILESIZE': '2000', 'rvm_bin_path': '/home/travis/.rvm/bin', 'DOCKER_HUB_USERNAME': 'fehguy', 'ELIXIR_VERSION': '1.0.4', 'RAILS_ENV': 'test', 'PERLBREW_ROOT': '/home/travis/perl5/perlbrew', 'TRAVIS_JOB_NUMBER': '7112.1', 'PYTHON_CONFIGURE_OPTS': '--enable-unicode=ucs4 --with-wide-unicode --enable-shared --enable-ipv6 --enable-loadable-sqlite-extensions --with-computed-gotos', 'TRAVIS_STACK_NAME': 'sugilite', 'TRAVIS_JOB_ID': '194540409', 'DERBY_HOME': '/usr/lib/jvm/java-9-oracle/db', 'TRAVIS_LANGUAGE': 'java', 'TRAVIS_BUILD_DIR': '/home/travis/build/swagger-api/swagger-codegen', 'HISTCONTROL': 'ignoredups:ignorespace', 'PWD': '/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python', 'TRAVIS_OS_NAME': 'linux', 'rvm_gem_options': '--no-ri --no-rdoc', 'rvm_remote_server_type4': 'rubies', 'XFILESEARCHPATH': '/usr/dt/app-defaults/%L/Dt'}

Processing ./.tox/dist/petstore_api-1.0.0.zip
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-WfmcLJ-build/setup.py", line 40, in <module>
        """
      File "/usr/lib/python2.7/distutils/core.py", line 111, in setup
        _setup_distribution = dist = klass(attrs)
      File "/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/local/lib/python2.7/site-packages/setuptools/dist.py", line 320, in __init__
        _Distribution.__init__(self, attrs)
      File "/usr/lib/python2.7/distutils/dist.py", line 287, in __init__
        self.finalize_options()
      File "/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/local/lib/python2.7/site-packages/setuptools/dist.py", line 386, in finalize_options
        ep.require(installer=self.fetch_build_egg)
      File "/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2306, in require
        items = working_set.resolve(reqs, env, installer)
      File "/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 853, in resolve
        raise VersionConflict(dist, req).with_context(dependent_req)
    pkg_resources.VersionConflict: (six 1.8.0 (/home/travis/build/swagger-api/swagger-codegen/samples/client/petstore/python/.tox/py27/lib/python2.7/site-packages), Requirement.parse('six>=1.10.0'))
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-WfmcLJ-build/
```
Ref: https://s3.amazonaws.com/archive.travis-ci.org/jobs/194540409/log.txt

There's been no change to the Python API client.

##### Swagger-codegen version

Latest master

##### Steps to reproduce

I could repeat the issue locally in my machine.

If I run `./test_python2.sh`, the tests pass locallly without issue. But if I run the following:
```
cd samples/client/petstore/python
virtualenv venv
source venv/bin/activate
./test_python2_and_3.sh
```
then I got the exact errors reported by the CI (travis)

**Comment by wing328** [comment]
@scottrw93 I wonder if you can take a look when you've time. I spent some time troubleshooting the issue but couldn't really figure out what went wrong.

**Comment by wing328** [comment]
Closed via #4643
